### PR TITLE
Reveal the initial resources if shroud is disabled.

### LINKED ISSUE #8943: Resources aren't revealed when starting a game with shroud disabled.
<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/25371656-resources-aren-t-revealed-when-starting-a-game-with-shroud-disabled?utm_campaign=plugin&utm_content=tracker%2F36085&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F36085&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by pchote**
Fixes #8943.

### CODE REVIEW

**Comment by penev92** [comment]
If you use the dev "Clear shroud" or reveal a big area by `MPStartLocations.InitialExploreRange` you still don't see resources (with shroud ON).
That feels inconsistent with what this PR does so IMO needs to be addressed.

**Comment by pchote** [comment]
Updated.

The regression was originally introduced in 7c0d3f4e4007226e23d2c9b39a594c6aedf341f3 (Lines 105-106).

File: OpenRA.Mods.Common/Traits/World/ResourceLayer.cs [hunk 1]
```diff
@@ -98,43 +119,24 @@ public void WorldLoaded(World w, WorldRenderer wr)
...
 					// Adjacent includes the current cell, so is always >= 1
 					var adjacent = GetAdjacentCellsWith(type, cell);
 					var density = int2.Lerp(0, type.Info.MaxDensity, adjacent, 9);
 					var temp = Content[cell];
 					temp.Density = Math.Max(density, 1);
 
-					Content[cell] = temp;
-					dirty.Add(cell);
+					// Initialize the RenderContent with the initial map state
+					// because the shroud may not be enabled.
```
Comment (Mailaender) [inline OpenRA.Mods.Common/Traits/World/ResourceLayer.cs]: > [SP2000] Invalid spacing at the end of the line.

**Comment by pchote** [comment]
Fixed.

**Comment by penev92** [comment]
Looks good :+1:

**Comment by Mailaender** [comment]
Works as promised. :white_check_mark:

**Pull request merged**